parse.h

```diff
@@ -117,7 +117,7 @@ static inline void *td_var(void *to, struct fio_option *o, unsigned int offset)
 	else
 		ret = to;
 
-	return (char *) ret + offset;
+	return ret + offset;
```

Comment:
Void pointer arithmetic is technically illegal in C (https://stackoverflow.com/a/3524270/9109338 ) and is a GCC extension...

Reply:
Yeah I do know that, but it's supported everywhere. I did ponder dropping this patch, but I'd be very surprised if I don't already do pointer math in a lot of other places in fio.

Reply:
Adding -Wpointer-arith gives me 103 warnings :-)